go/go.mod

```diff
@@ -0,0 +1,26 @@
+module ginrest
+
+go 1.16
+
+require (
+	github.com/bitly/go-simplejson v0.5.1
+	github.com/gin-contrib/gzip v0.0.1
+	github.com/gin-gonic/gin v1.7.2
+	github.com/go-gorp/gorp v2.2.0+incompatible
+	github.com/go-playground/validator/v10 v10.8.0
+	github.com/go-sql-driver/mysql v1.7.1
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/google/uuid v1.3.0
+	github.com/joho/godotenv v1.3.0
+	github.com/json-iterator/go v1.1.11 // indirect
+	github.com/lib/pq v1.5.2 // indirect
```

Comment:
Why do we have a postgres driver here?

Reply:
Sorry, it's my mistake.
I will remove it.

Reply:
@danfowler 
remove it.
https://github.com/omise/examples/pull/32/commits/d2f4b3d3f976cf2c3561adbfab53bd4ae02918eb